process/sandbox.md

```diff
@@ -0,0 +1,99 @@
+# CNCF Sandbox Guidelines v1.0
+![CNCF Sandbox](https://github.com/cncf/artwork/blob/master/other/cncf-sandbox/horizontal/color/cncf-sandbox-horizontal-color.png)
```

Comment:
As expressed in the mailing list, please consider the name. Sandbox has a lot of bad baggage and connotations that undermine the intent. Please consider other names for this stage. My two humble suggestions are (my apologies if this has been proposed already) Launchpad or Runway.